pkgs/top-level/all-packages.nix

```diff
@@ -18947,6 +18947,9 @@ in
 
   sogo = callPackage ../servers/web-apps/sogo { };
 
+  spacecookie =
+    haskell.lib.justStaticExecutables haskellPackages.spacecookie;
```

Comment:
This newline is not required.

Reply:
I don't understand why you keep pointing out formatting “issues” on already merged PRs. We have no established formatting guideline in the first place and especially with merged PRs this serves really nothing other than wasting the time of everyone involved.

Reply:
> I don't understand why you keep pointing out formatting “issues” on already merged PRs.

If I have 50 tabs open GitHub sometimes just forgets to refresh them and they are only updated after I posted a comment.

> We have no established formatting guideline in the first place

Just do reasonable formatting and that new line there is just because nixfmt enforces a 80 line length which we are not following in nixpkgs anyway.